extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -194,6 +194,10 @@ tags: {  # <amp-video> in amp-story
     name: "cache"
     value: "google"
   }
+  attrs: {
+    name: "captions-id"
+    requires_extension: "amp-story-captions"
```

Comment:
As a followup, I think we should make it either optional and forbidden, and lazy load this extension.